app/src/main/java/fr/free/nrw/commons/settings/Prefs.java

```diff
@@ -7,8 +7,8 @@
     public static final String DEFAULT_LICENSE = "defaultLicense";
 
     public static class Licenses {
-        public static final String CC_BY_SA_3 = "CC BY-SA";
-        public static final String CC_BY_3 = "CC BY";
+        public static final String CC_BY_SA_3 = "CC BY-SA 3.0";
+        public static final String CC_BY_3 = "CC BY 3.0";
```

Comment:
Should have made it clearer, I changed it to match strings.xml and therefore arrays.xml otherwise you'd get mismatches on those settings in singleuploadfragment